metricbeat/module/mongodb/dbstats/_meta/fields.yml

```diff
@@ -0,0 +1,56 @@
+- name: dbstats
+  type: group
+  description: >
+    dbstats provides an overview of a particular mongo database. This document
+    is most concerned with data volumes of a database.
+  fields:
+    - name: avg_obj_size
+      type: long
+
+    - name: collections
+      type: integer
+
+    - name: data_size
+      type: long
+
+    - name: db
+      type: keyword
+
+    - name: file_size
+      type: long
+
+    - name: index_size
```

Comment:
this is probably `index_size.bytes` and then you can add `format: bytes` to each block, so Kibana will automatically now that it is in bytes and can do some conversion.